Source/ASTextNode.mm

```diff
@@ -136,6 +136,7 @@ @interface ASTextNodeDrawParameter : NSObject {
 @package
   ASTextKitAttributes _rendererAttributes;
   UIColor *_backgroundColor;
+  UIColor *_tintColor;
```

Comment:
I think we'll actually undo all the changes to this file since the "tint" is handled from ASButtonNode right now